src/Illuminate/Database/DatabaseManager.php

```diff
@@ -280,7 +280,25 @@ public function supportedDrivers()
      */
     public function availableDrivers()
     {
-        return array_intersect($this->supportedDrivers(), PDO::getAvailableDrivers());
+        $alternateDriverMap = [
```

Comment:
I made it an array so it's easy to modify if need be, and the name is so it's obvious what it is exactly. But the whole thing could definitely be made more compact without a set of replacements.